app/logic/solana_logic.rb

```diff
@@ -495,8 +497,12 @@ def cli_request(cli_method, rpc_urls)
       end
 
       # byebug
-
-      return JSON.parse(response_utf8) unless response_utf8 == ''
+      if response_utf8 != ''
+        # Include errors from the solana-cli 
+        return JSON.parse(response_utf8).merge!('cli_error' => error_msg)
```

Comment:
Remember that Ruby has implicit return for whatever is evaluated last, so the `return` isn't needed and can be removed.